dev_swift/02b_initializing.ipynb

```diff
@@ -0,0 +1,404 @@
+{
```

Comment:
<p>M is a scalar, so I think the original PyTorch expression is right (i.e., `matmul(x, A).norm &lt;= M * x.norm()`)</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Yes M is a constant number here.